### DISCUSSION TITLE
The number of training iterations for the generator and discriminator.

### DISCUSSION THREAD

**Discussion by andyye1999** [jhauret/eben#6] [Ideas]
hi jhauret 

during the training process and in the paper, there is a significant difference in parameter quantity between the generator and discriminator. Should we train the generator multiple times before training the discriminator once? Is the increase in loss function of the generator due to insufficient generating ability to cope with the discriminator caused by this parameter difference?
